mod/wikirate_source/set/type/source/source_preview.rb

```diff
@@ -159,131 +143,15 @@ def valid_mime_type? mime_type
     )
   end
 
-  # View: HTML for the navigation bar on preview page
-  view :navigation_bar, tags: :unknown_ok  do |args|
-    navbar_brand = nest(Card[:logo], view: :core, size: :original)
-    %(
-      <nav class="navbar navbar-default  ">
-
-        <div class="">
-          <!-- Brand and toggle get grouped for better mobile display -->
-          <div class="navbar-header">
-            <button type="button" class="navbar-toggle collapsed"
-                    data-toggle="collapse"
-                    data-target="#bs-example-navbar-collapse-1">
-              <span class="sr-only">Toggle navigation</span>
-              <span class="icon-bar"></span>
-              <span class="icon-bar"></span>
-              <span class="icon-bar"></span>
-            </button>
-            <div id="source-preview-tabs" class="navbar-brand" href="#">
-              #{link_to_resource '/', raw(navbar_brand)}
-            </div>
-          </div>
-
-
-
-          <!-- Collect the nav links, forms, and other content for toggling -->
-          <div class="collapse navbar-collapse"
-               id="bs-example-navbar-collapse-1">
-            <!-- Navbar Menu -->
-            #{subformat(Card['nav_bar_menu']).render_content}
-            <ul class="nav nav-tabs navbar-right gray-color ">
-               #{render_preview_options(args)}
-            </ul>
-          </div>
-          <!-- /.navbar-collapse -->
-        </div>
-        <!-- /.container-fluid -->
-      </nav>
-
-    )
-  end
-
   view :non_previewable, tags: :unknown_ok do |_args|
     if file_card = Card[card.name + "+File"]
       <<-HTML
         <a href="#{file_card.attachment.url}" class="btn btn-primary" role="button">Download</a>
       HTML
     else
-      url_card = card.fetch(trait: :wikirate_link)
-      url = url_card ? url_card.item_names.first : nil
       <<-HTML
         <a href="#{preview_url}" class="btn btn-primary" role="button">Visit Original Source</a>
       HTML
     end
   end
```

Comment:
since you refactored/removed all that other stuff no reason to stop here and save that ugly piece
For example:
```
file_card = card.fetch trait: :file
url, text = file_card ? [file_card.attachment.url, "Download"] : [preview_url, "Visit Original Source"]
link_to text, href: url, class: "btn btn-primary", role: "button"
```